pkg/hostagent/opflex.go

```diff
@@ -86,6 +86,17 @@ func (agent *HostAgent) discoverHostConfig() (conf *HostAgentNodeConfig) {
 		conf.OpflexPeerIp = "127.0.0.1"
 		agent.log.Debug("\n  == Opflex: Running in overlay mode ==\n")
 		return
+	} else if agent.config.OpflexMode == "dpu" {
+		conf = &HostAgentNodeConfig{}
+		conf.VxlanIface = "bond0.4093"
+		conf.UplinkIface = "bond0"
+		conf.VxlanAnycastIp = "10.0.0.32"
+		conf.OpflexPeerIp = "10.0.0.30"
```

Comment:
Why are the above values hardcoded, would they not be system dependent?

Reply:
That is hardcoded because the bond interface is not present on the host. Its present on the DPU and again a customer cannot provision this currently because he has no access to DPU. I thought these do not change since I have never seen a different value for them in any config. But I might be wrong. Again the DPU side has its own config thats missing in this review. One way to make these dynamic would be to get them from the k8s setup we do on the DPU. There are 2 k8s clusters running, one on DPU that we control and another on host. Thanks.

Reply:
But its a more involved change because it involves.
1. DPU using whatever from the k8s config it received.
2. Host agent sending a remote ip link command to dpu and populating those values.
3
Technically I do not see why the host agent should even care what those values are besides the sanity check. But today if that isnt populated host agent will fail to come up. So if thats the case we can do the sanity check on DPU and skip 2 that is the more involved change.

Reply:
Thanks Madhu. Assuming that the 4093 here corresponds to the infra VLAN, that could definitely vary from one setup to another. Anyway, I will merge these changes for now with the understanding that a follow up PR for DPU changes is on the way and can provide any changes if required.